tests/unit-tests/test-class-quiz.php

```diff
@@ -1723,4 +1723,41 @@ public function testResetUserLessonData_WhenCalled_ResetsTheQuizGrades() {
 		/* Assert. */
 		$this->assertEmpty( $quiz_grades );
 	}
+
+	public function testCreatingQuizSetsAuthorToLessonTeacher() {
```

Comment:
Ahh I'll give that a read. I've been writing a lot of unit tests in my PR's, and it will be nice to have some consistency. I'll have a look and make some changes. Good catch!